region/maridia/inner-yellow.json

```diff
@@ -907,8 +939,79 @@
                     "Then morph on top and damage boost into the morph tunnel as it unfreezes."
                   ]
                 }
-              ],
-              "devNote": "FIXME Menus are tricky to deal with with Power Beam alone, information should be added to kill or avoid them, likely while adding a third node."
+              ]
+            }
+          ]
+        },
+        {
+          "from": 3,
+          "to": [
+            {
+              "id": 1,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "HiJump",
+                      "SpaceJump",
+                      "canTrickyDashJump",
+                      "canTrickyUseFrozenEnemies",
+                      "canPreciseWalljump"
+                    ]}
+                  ],
+                  "devNote": "The menus don't really matter with these strats."
+                },
+                {
+                  "name": "Kill the Menus",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "h_canIBJ",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir"
+                    ]},
+                    {"or":[
+                      {"enemyKill":{
+                        "enemies": [ [ "Menu", "Menu", "Menu" ] ],
+                        "explicitWeapons": [
```

Comment:
Missile kill is my first choice.

Reply:
Good catch. I was thinking this would be harder to aim. I guess it is really simple though